src/ncp/ncp_base.cpp

```diff
@@ -239,6 +241,9 @@ const NcpBase::SetPropertyHandlerEntry NcpBase::mSetPropertyHandlerTable[] =
     { SPINEL_PROP_MAC_SCAN_PERIOD, &NcpBase::SetPropertyHandler_MAC_SCAN_PERIOD },
     { SPINEL_PROP_MAC_15_4_PANID, &NcpBase::SetPropertyHandler_MAC_15_4_PANID },
     { SPINEL_PROP_MAC_RAW_STREAM_ENABLED, &NcpBase::SetPropertyHandler_MAC_RAW_STREAM_ENABLED },
+#if OPENTHREAD_ENABLE_RAW_LINK_API
+    { SPINEL_PROP_STREAM_RAW, &NcpBase::SetPropertyHandler_STREAM_RAW },
+#endif // OPENTHREAD_ENABLE_RAW_LINK_API
```

Comment:
I am ok with both ways.
I like combining them maybe a bit more (to keep the code cleaner)...